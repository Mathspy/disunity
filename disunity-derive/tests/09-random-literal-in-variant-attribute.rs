use disunity_derive::Variant;

#[derive(Variant)]
enum Class {
    #[disunity("hi from random valley")]
    Magic(i32),
}

#[derive(Variant)]
enum Other {
    #[disunity(discriminant = 4, "hi from random valley")]
    Magicks(i32),
}

fn main() {}
