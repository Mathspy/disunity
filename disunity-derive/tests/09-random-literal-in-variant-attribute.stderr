error: unexpected literal in disunity attribute macro
 --> tests/09-random-literal-in-variant-attribute.rs:5:16
  |
5 |     #[disunity("hi from random valley")]
  |                ^^^^^^^^^^^^^^^^^^^^^^^

error: unexpected literal in disunity attribute macro
  --> tests/09-random-literal-in-variant-attribute.rs:11:34
   |
11 |     #[disunity(discriminant = 4, "hi from random valley")]
   |                                  ^^^^^^^^^^^^^^^^^^^^^^^
